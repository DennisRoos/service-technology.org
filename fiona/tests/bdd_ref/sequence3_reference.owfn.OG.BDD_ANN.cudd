.ver DDDMP-2.0
.mode A
.varinfo 4
.dd bddAnn
.nnodes 8
.nvars 7
.nsuppvars 7
.suppvarnames !a !b !c final 0 1 2
.orderedvarnames !b !c !a final 0 1 2
.ids 0 1 2 3 4 5 6
.permids 2 0 1 3 4 5 6
.nroots 1
.rootids 8
.nodes
1 1 0 0
2 6 1 -1
3 5 2 1
4 4 3 1
5 3 4 1
6 2 1 -5
7 1 1 6
8 0 1 7
.end
