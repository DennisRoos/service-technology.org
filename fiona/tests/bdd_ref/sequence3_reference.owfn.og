INTERFACE
  INPUT
;
  OUTPUT
    a,
    b,
    c;

NODES
  0 : (!a + !b + !c) : blue,
  1 : (!a + !b) : blue,
  2 : !a : blue,
  3 : final : blue : finalnode,
  4 : !b : blue,
  5 : (!a + !c) : blue,
  6 : !c : blue,
  7 : (!b + !c) : blue;

INITIALNODE
  0;

TRANSITIONS
  0 -> 1 : !c,
  0 -> 5 : !b,
  0 -> 7 : !a,
  1 -> 2 : !b,
  1 -> 4 : !a,
  2 -> 3 : !a,
  4 -> 3 : !b,
  5 -> 2 : !c,
  5 -> 6 : !a,
  6 -> 3 : !c,
  7 -> 4 : !c,
  7 -> 6 : !b;
