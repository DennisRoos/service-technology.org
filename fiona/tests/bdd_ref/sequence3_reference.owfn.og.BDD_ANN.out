digraph "DD" {
size = "7.5,10"
center = true;
edge [dir = none];
{ node [shape = plaintext];
  edge [style = invis];
  "CONST NODES" [style = invis];
" 0 " -> " 1 " -> " 2 " -> " 3 " -> " 4 " -> " 5 " -> " 6 " -> "CONST NODES"; 
}
{ rank = same; node [shape = box]; edge [style = invis];
"F0"; }
{ rank = same; " 0 ";
"58";
}
{ rank = same; " 1 ";
"b5";
"57";
}
{ rank = same; " 2 ";
"b4";
"55";
"51";
"b1";
}
{ rank = same; " 3 ";
"8c";
"50";
"4d";
"b0";
"72";
"54";
"ae";
"b3";
}
{ rank = same; " 4 ";
"8a";
"b2";
"53";
"af";
"4e";
"71";
"4c";
"8b";
"98";
"77";
"4f";
"52";
}
{ rank = same; " 5 ";
"91";
"76";
"68";
"56";
"79";
"6d";
}
{ rank = same; " 6 ";
"67";
}
{ rank = same; "CONST NODES";
{ node [shape = box]; "43";
}
}
"F0" -> "58" [style = solid];
"58" -> "b5";
"58" -> "57" [style = dashed];
"b5" -> "b1";
"b5" -> "b4" [style = dashed];
"57" -> "51";
"57" -> "55" [style = dashed];
"b4" -> "b3";
"b4" -> "8c" [style = dotted];
"55" -> "54";
"55" -> "72" [style = dotted];
"51" -> "4d";
"51" -> "50" [style = dashed];
"b1" -> "ae";
"b1" -> "b0" [style = dashed];
"8c" -> "8a";
"8c" -> "8b" [style = dashed];
"50" -> "4e";
"50" -> "4f" [style = dashed];
"4d" -> "4c";
"4d" -> "6d" [style = dashed];
"b0" -> "98";
"b0" -> "af" [style = dashed];
"72" -> "71";
"72" -> "43" [style = dashed];
"54" -> "52";
"54" -> "53" [style = dashed];
"ae" -> "43";
"ae" -> "71" [style = dashed];
"b3" -> "77";
"b3" -> "b2" [style = dashed];
"8a" -> "79";
"8a" -> "67" [style = dashed];
"b2" -> "6d";
"b2" -> "76" [style = dashed];
"53" -> "67";
"53" -> "91" [style = dashed];
"af" -> "6d";
"af" -> "79" [style = dotted];
"4e" -> "76";
"4e" -> "56" [style = dotted];
"71" -> "6d";
"71" -> "43" [style = dashed];
"4c" -> "43";
"4c" -> "6d" [style = dashed];
"8b" -> "56";
"8b" -> "67" [style = dashed];
"98" -> "43";
"98" -> "79" [style = dotted];
"77" -> "43";
"77" -> "76" [style = dashed];
"4f" -> "91";
"4f" -> "56" [style = dotted];
"52" -> "68";
"52" -> "91" [style = dashed];
"91" -> "67";
"91" -> "67" [style = dotted];
"76" -> "43";
"76" -> "67" [style = dotted];
"68" -> "43";
"68" -> "67" [style = dashed];
"56" -> "43";
"56" -> "43" [style = dotted];
"79" -> "67";
"79" -> "43" [style = dotted];
"6d" -> "67";
"6d" -> "43" [style = dashed];
"67" -> "43";
"67" -> "43" [style = dotted];
"43" [label = "1"];
}
